kernel/build-kernel.sh

```diff
@@ -365,8 +383,17 @@ install_kata() {
 	config_version=$(get_config_version)
 	[ -n "${config_version}" ] || die "failed to get config version"
 	install_path=$(readlink -m "${DESTDIR}/${PREFIX}/share/${project_name}")
-	vmlinuz="vmlinuz-${kernel_version}-${config_version}"
-	vmlinux="vmlinux-${kernel_version}-${config_version}"
+
+	sufix=""
```

Comment:
nit - I think you inherited this, but - s/sufix/suffix/

Reply:
fixed